textile/features.textile

```diff
@@ -666,6 +668,7 @@ h3(#realtime-channel). RealtimeChannel
 ** @(RTL7d)@ Messages delivered are automatically decoded based on the @encoding@ attribute; see @RestChannel@ encoding features. Tests should exist to publish and subscribe to encoded messages using the "AES 128":https://github.com/ably/ably-common/blob/main/test-resources/crypto-data-128.json and "AES 256":https://github.com/ably/ably-common/blob/main/test-resources/crypto-data-256.json fixture test data
 ** @(RTL7e)@ If a message cannot be decoded or decrypted successfully, it should be delivered to the listener with the @encoding@ attribute set indicating the residual encoding state, and an error should be logged
 ** @(RTL7f)@ A test should exist ensuring published messages are not echoed back to the subscriber when @echoMessages@ is set to false in the @RealtimeClient@ library constructor
+** @(RTL7g)@ Optionally, upon success, may return the @ChannelStateChange@ object once the channel is attached. If the channel is already attached, a synthetic @ChannelStateChange@ from @ATTACHED@ to @ATTACHED@ may be returned.
```

Comment:
This should probably be written with reference to the "optional callback" instead of the return value since that's the language used in RTL7c. Also I think it needs updating to match the changed wording for the corresponding `#attach` spec point.